services/external_sources/external_sources.py

```diff
@@ -11,6 +11,7 @@
 logger = logging.getLogger(__name__)
 load_dotenv()
 
+DEFAULT_SEARCH_TERM = "world population"
 
```

Comment:
two newlines between function definitions and other content such as a constant. you can check with `flake8 services` to see any of those issues